packages/venia-ui/lib/components/FormError/formError.js

```diff
@@ -11,6 +12,25 @@ import defaultClasses from './formError.css';
 const FormError = props => {
     const { classes: propClasses, errors, scrollOnError } = props;
 
+    const { formatMessage } = useIntl();
+
+    var graphQLErrorMessage;
```

Comment:
We don't use `var` in JavaScript https://dev.to/mindninjax/stop-using-var-for-declaring-variables-2p3a